Cargo.toml

```diff
@@ -21,6 +22,11 @@ pkg-config = { version = "0.3", optional = true }
 chrono = "0.2"
 nom = "2.0"
 systemstat = { version = "0", optional = true }
-xcb = { version = "0.7", features = ["xkb"], optional = true }
+xcb = { version = "0.8", features = ["xkb"], optional = true }
 serde = "0.8"
 serde_json = "0.8"
+alsa = { version = "0.1.10", optional = true }
+libc = { version = "0.2.29", optional = true }
+
+[patch.crates-io]
```

Comment:
hmm what's this?

Reply:
again patch.crates-io snuck in unintentionally, I will remove it.

Reply:
But what *is* the `patch` thing? Never seen that before :)

Reply:
Afaik, it lets me temporarily change the source of some dependency. Specifically I needed linux cpu support from `systemstat` on github (perhaps there should be a new release?). I don't really remember why options that went into .cargo/config didn't work for me.